_episodes/01-introduction.md

```diff
@@ -79,16 +81,23 @@ instruction-intensive operations in order to improve the overall performance
 compared with both sequential and homogeneous parallel programming models.
 
 To better understand the concept of heterogeneous parallel programming,
-we should introduce the following concepts:
+we will introduce the following concepts from computer networking:
 
-- **Latency**: The duration of an operation from its beginning to its completion expressed in microseconds (*m*s)
-- **Throughput**: The number of operations processed per unit of time expressed in gigaflops (*G*flops), which
-translates into a billion floating-point operations per second.
-- **Bandwidth**: The processed amount of data per unit of time expressed in megabytes per second (*M*B/s) 
-or gigabytes per second (*G*B/s)
+- **Latency**: The duration of an operation from its beginning to its completion expressed in units of time e.g. microseconds (*m*s)
+- **Bandwidth**: The *theoretical* capacity of how much data could be transferred from source to destination per unit of time,
+  commonly expressed in gigabytes per second (*G*B/s) for GPUs  
+- **Throughput**: The *real* measure of how much data is successfully transferred from source to destination per unit of time, 
+expressed in gigaflops (*G*flops) for GPUs, which translates into a billion floating-point operations per second. Therefore, throughput 
+is always less than the bandwidth.
+
+![Figure 0](../fig/pipe.png)
```

Comment:
Like the simple idea that this figure tries to convey. However, did you make it or borrowed it from somewhere.

Reply:
Pretty sure I got it from somewhere and edited the hell out of it over the years, will try to find its source ... or just draw a pipe flow.

Reply:
Don't bother. I will work on it to create something like it either in Photoshop or Gimp